staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -762,8 +763,10 @@ func DefaultBuildHandlerChain(apiHandler http.Handler, c *Config) http.Handler {
 	handler = genericapifilters.WithWarningRecorder(handler)
 	handler = genericapifilters.WithCacheControl(handler)
 	handler = genericfilters.WithHSTS(handler, c.HSTSDirectives)
+	handler = httplog.WithLogging(handler, httplog.DefaultStacktracePred)
```

Comment:
why have we moved it here?

Reply:
good catch, this was an artifact for another follow up PR. I remove it.